web/translation/translate.en_US.toml

```diff
@@ -32,7 +32,7 @@
 "transmission" = "Transmission"
 "host" = "Host"
 "path" = "Path"
-"camouflage" = "Camouflage"
+"camouflage" = "Obfuscation"
```

Comment:
These two terms are not same.
Camouflage was used in first version of X-UI due to better understanding.
If you want to keep it, please change it in another languages.